nrf_wifi/fw_if/umac_if/inc/fmac_util.h

```diff
@@ -134,6 +134,12 @@ bool nrf_wifi_util_is_arr_zero(unsigned char *arr,
 
 #endif /* !CONFIG_NRF700X_RADIO_TEST */
 
+#ifdef CONFIG_NRF700X_RAW_DATA_TX
```

Comment:
Please relocate these to fmac_api.h since we are expecting these to be invoked by the shim layer code (`fmac_utli.h` is intended as utility functions for the FMAC layer in the OSAL code)

Reply:
okay

Reply:
done